syntax = "proto3"

message Policy {
    unint32 policyNumber = 1;
    double policyPrice = 2;
    string policyDetails = 3;
}
